mllib/src/main/scala/org/apache/spark/mllib/stat/Statistics.scala

```diff
@@ -212,4 +213,15 @@ object Statistics {
     : KolmogorovSmirnovTestResult = {
     KolmogorovSmirnovTest.testOneSample(data, distName, params: _*)
   }
+
+  /** Java-friendly version of [[kolmogorovSmirnovTest()]] */
+  @varargs
+  def kolmogorovSmirnovTest(
+      data: JavaRDD[java.lang.Double],
+      distName: String,
+      params: java.lang.Double*): KolmogorovSmirnovTestResult = {
+    val javaParams = params.toSeq.asInstanceOf[Seq[Double]]
```

Comment:
I have the same question wrt RDD's . For example when one does rdd.asInstanceOf[RDD(java.lang.Long)]